src/cli.spec.ts

```diff
@@ -163,58 +160,12 @@ describe('cli', () => {
         });
     });
 
-    it('Retrieves a signed package', async () => {
```

Comment:
Can we restore these tests and make them work with how the CLI runs? Or move them to a different file and test the ExecCommand instead?

Reply:
Decided not to reimplement